packages/communication-react/review/communication-react.api.md

```diff
@@ -1083,6 +1078,11 @@ export type DisplayNameChangedListener = (event: {
     displayName: string;
 }) => void;
 
+// @public
+export interface Disposal {
```

Comment:
Are you sure that the suggested naming was `Disposal`? I've seen variants of `Disposable` for this.